server.js

```diff
@@ -7,13 +7,42 @@ import mongoose from "mongoose";
 // import avocadoSalesData from "./data/avocado-sales.json";
 // import booksData from "./data/books.json";
 // import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
+import netflixData from "./data/netflix-titles.json"
 // import topMusicData from "./data/top-music.json";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || "mongodb://127.0.0.1/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
+const { Schema } = mongoose;
+
+const netflixTitleSchema = new Schema({
+  id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String
+});
+
+const NetflixTitle = mongoose.model('NetflixTitle', netflixTitleSchema);
+
+if (process.env.RESET_DB) { 
```

Comment:
you can also make an endpoint that will be resetting the DB instead of writing in the terminal: 
```
app.get("/reset", async (req, res) => {
  try {
    await resetDatabase();
    res.json({
      success: true
    }) 
  } catch (e) {
    res.json({
      success: false,
    })
  }
})
```